dbt/parsely_dpl_redshift/redshift.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import absolute_import
+
+import datetime as dt
+import argparse
+import subprocess
+import os
+
+import parsely_raw_data.redshift as prd_redshift
+import parsely_raw_data.utils as utils
+
+def parse_datetime_arg(arg):
```

Comment:
This might be a good function to stick in `parsely_raw_data.utils` and import, as it could likely end up being used elsewhere. I would use a separator as well, i.e. YYYY-MM-DD